src/Valit/Errors/ValitRuleError.cs

```diff
@@ -1,21 +1,25 @@
+using System;
+
 namespace Valit.Errors
 {
     internal class ValitRuleError
     {
-        public string Message  { get; }
+        private readonly Func<string> _message;
```

Comment:
can you add "Func" sufix, just for consistency ?

Reply:
It has been done.